components/Chat/Chat.vue

```diff
@@ -524,18 +526,26 @@
         // Remove trolls
         if ( this.hideTrolls && message.username.startsWith( 'troll:' ) ) return true;
 
+        // Add username highlighting
+        message.message = message.message.replace( pattern, `<span class="highlight">$&</span>` );
```

Comment:
A bit counter-intuitive, and forces these checks to be last. But code that works > good code that doesn't.

Reply:
I suppose instead of returning early it could set a flag. Next person who touches that function can take a stab at refactoring it to read nicer.